### ISSUE TITLE
Syntax error when generating code on Informix 12.10 without DELIMIDENT=y

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#8417]
It appears that on older Informix versions, the `"informix"."systables"` table cannot be accessed when quoted:

```
[WARNING] SQL exception            : Exception while executing meta query: A syntax error has occurred.
select distinct trim("informix"."systables"."owner")
from "informix"."systables"
where "informix"."systables"."owner" in ('<schema name>')

[ERROR] Could not load schemata  
org.jooq.exception.DataAccessException: SQL [select distinct trim("informix"."systables"."owner") from "informix"."systables" where "informix"."systables"."owner" in (cast(? as lvarchar)) -- SQL rendered with a free trial version of jOOQ 3.11.9]; A syntax error has occurred.
     at org.jooq_3.11.9.INFORMIX.debug(Unknown Source)
     at org.jooq.impl.Tools.translate(Tools.java:2429)
```

The query works when the quotes in the from clause are removed, e.g.

```sql
select distinct trim("informix"."systables"."owner")
from informix.systables
where "informix"."systables"."owner" in ('<schema name>')```
```

A workaround is to provide settings to the code generator, e.g. through the classpath at `/jooq-settings.xml`, or via the `-Dorg.jooq.settings` system property

**Comment by lukaseder** [comment]
The workaround here is to make sure `DELIMIDENT=y` is placed in the JDBC URL to make informix support double quoted identifiers.

The fix will check if `"systables"` is available, and if not, turn off quoting of identifiers in the code generator.

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.12. Backport scheduled to 3.11.11 (#8444)
